README.md

```diff
@@ -48,7 +48,7 @@ This will launch the interactive prompt. Type `help` to get a list of commands.
 ## Notes:
 * If you want to run your own server, please see [zingo lightwalletd](https://github.com/zingolabs/lightwalletd), and then run `./zingo-cli --server http://127.0.0.1:9067`
 * The default log file is in `~/.zcash/zingo-wallet.debug.log`. A default wallet is stored in `~/.zcash/zingo-wallet.dat`
-* Currently, the default, hard-coded `lightwalletd` server is https://lwdv3.zecwallet.co:443/. To change this, you can modify the `DEFAULT_SERVER` const in `config/src/lib.rs`
+* Currently, the default, hard-coded `lightwalletd` server is https://zuul.free2z.cash:9067/. To change this, you can modify the `DEFAULT_SERVER` const in `config/src/lib.rs`
```

Comment:
mainnet.lightwalletd.com should on paper be the most reliable but it hasn't been recently. I think being able to configure and easily switch (ENV variable?) should be high priority feature.

Reply:
This seems sufficient to me.   I am +1 on a merge.  I cannot merge, since I opened it.   Can someone please merge?

Reply:
Nope, no way to change the server in the UI mobile